prefix.go

```diff
@@ -484,6 +484,11 @@ func (i *ipamer) newPrefix(cidr, parentCidr string) (*Prefix, error) {
 	return p, nil
 }
 
+// Retrieve all existing Prefix CIDRs from the underlying storage
```

Comment:
same here

Reply:
still missing here

Reply:
Sorry about that, forgot it was in two places.  Should be all good now